libnetwork/internal/netiputil/netiputil.go

```diff
@@ -35,14 +35,14 @@ func toPrefix(n *net.IPNet) (netip.Prefix, bool) {
 	return netip.PrefixFrom(addr.Unmap(), ones), true
 }
 
-func hostID(addr netip.Addr, bits uint) uint64 {
+func HostID(addr netip.Addr, bits uint) uint64 {
 	return ipbits.Field(addr, bits, uint(addr.BitLen()))
 }
 
 // subnetRange returns the amount to add to network.Addr() in order to yield the
```

Comment:
Looks like the function is now exported, but the GoDoc wasn't updated accordingly;

```suggestion
// SubnetRange returns the amount to add to network.Addr() in order to yield the
```

Reply:
Fixed - and added GoDoc comments to the other functions in the file.